locale/manageiq.pot

```diff
@@ -27045,6 +27056,16 @@ msgstr ""
 msgid "EVM Zone"
 msgstr ""
 
+#. TRANSLATORS: file: product/dashboard/widgets/rss_newest_hosts.yaml
```

Comment:
Do we even have RSS widgets anymore?

Reply:
Added to https://github.com/ManageIQ/manageiq/issues/21377 as https://github.com/ManageIQ/manageiq/issues/22778